data/json/effects.json

```diff
@@ -2077,6 +2077,44 @@
     "rating": "bad",
     "base_mods": { "vomit_chance": [ 500 ] }
   },
+  {
+    "type": "effect_type",
+    "id": "bile_stink",
+    "name": [ "Bile Stink" ],
+    "desc": [ "You're splattered with a greasy layer of liquified putrescence." ],
+    "apply_message": "You're covered in a foul smelling fluid!",
+    "decay_messages": [ [ "The rancid smell of corpse bile finally goes away.", "good" ] ],
+    "rating": "bad",
+    "resist_traits": [ "SAPROVORE", "EATDEAD" ],
+    "base_mods": { "vomit_chance": [ 500, 2000 ] },
```

Comment:
```suggestion
    "base_mods": { "vomit_chance": [ 500, 2000 ] },
    "enchantments": [ { "values": [ { "value": "SCENT_MASK", "add": 500 } ] } ],
```

Reply:
Someone should migrate this enchantment name to zombie_attractant or something better named than SCENT_MASK